### ISSUE TITLE
transformCounts error

### ISSUE THREAD

**Issue by kusandeep** [microbiome/miaViz#47]
Hi, 
Thanks for creating such an awesome package.
I am getting an error at transformCounts function, must be very basic mistake done by me, but would be great if I can get some help.

'> v5v7 <- transformCounts(v5v7, method = "relabundance") 
Error in MatrixGenerics:::.load_next_suggested_package_to_search(x) : 
  Failed to find a colSums2() method for data.frame objects.'

Thaks and much appreciated for help

**Comment by antagomir** [comment]
What is the class of the `v5v7` object?

**Comment by TuomasBorman** [comment]
Hi!

The problem is that your assay is in data.frame format; assays should be in matrix format. 

Does this work:

**Comment by kusandeep** [comment]
Yes. All working now. Thanks for the quick response, loving the package so far. 
I probably will come up with some more queries :).

**Comment by TuomasBorman** [comment]
OK, good. All feedback is welcome, and we are happy to help if something is unclear

-Tuomas
